go/worker/compute/executor/committee/node.go

```diff
@@ -982,27 +1017,26 @@ func (n *Node) startProcessingBatchLocked(batch *unresolvedBatch) {
 	blk := n.commonNode.CurrentBlock
 	consensusBlk := n.commonNode.CurrentConsensusBlock
 	height := n.commonNode.CurrentBlockHeight
+	roundResults := lastRoundResults
 	go func() {
 		defer close(done)
 
-		// Fetch message results emitted during the last normal round.
-		state, err := n.commonNode.Consensus.RootHash().GetRuntimeState(ctx, blk.Header.Namespace, height)
+		rtDesc, err := n.commonNode.Runtime.ActiveDescriptor(n.ctx)
```

Comment:
Hm actually I think even doing it like this is not enough as this doesn't guarantee that what is queried here is synced with `height`. Would be better to just do a `GetRuntimeState` query here as before.